packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -66,29 +73,20 @@ function MyFeedHeading({
         ? AlertPlacement.Right
         : AlertPlacement.Bottom,
   };
-
-  if (searchVersion === SearchExperiment.V1) {
-    return (
-      <AlertPointer {...alertProps} offset={[0, 0]}>
-        <Button
-          variant={ButtonVariant.Float}
```

Comment:
The only difference is the variant and offset, no need for us to have the whole block of code.